src/main/java/com/pi4j/drivers/input/rotary/adafruit5880/Adafruit5880Driver.java

```diff
@@ -213,18 +199,18 @@ public void onDigitalStateChange(DigitalStateChangeEvent event) {
 
             if (event.state() == DigitalState.LOW) {
                 // Only way to clear the interupt is to read the position
-                if (lastKnownPosition != driver.getPosition()) {
-                    lastKnownPosition = driver.getPosition();
+                if (lastKnownPosition != Adafruit5880Driver.this.getPosition()) {
```

Comment:
This is one reason why I don't use "this" typically if there is no ambiguity... :)

I'd change this to 

```
int newPosition = getPosition()
if (newPosition != lastKnownPosition) {
  lastKnownPosition = newPosition;
  ...
```

The reason is that the position might change between all three calls -- and it really should only be necessary to make a single call to obtain the value the interrupt was for....